src/js/actions/mask.js

```diff
@@ -142,6 +150,98 @@ define(function (require, exports) {
             menuActions.native]
     };
 
+    /**
+     * Create an circular mask the size of the currently selected layer
```

Comment:
I was gonna say the comment has a typo, then I realized it doesn't even describe the function correctly.